src/paulscode/android/mupen64plusae/GalleryActivity.java

```diff
@@ -177,7 +181,7 @@ protected void onCreate( Bundle savedInstanceState )
         
         // Configure the navigation drawer
         mDrawerLayout = (DrawerLayout) findViewById( R.id.drawerLayout );
-        mDrawerToggle = new ActionBarDrawerToggle( this, mDrawerLayout, toolbar, R.string.app_name, R.string.app_name )
+        mDrawerToggle = new ActionBarDrawerToggle( this, mDrawerLayout, toolbar, 0, 0 )
```

Comment:
Accidentally had the label for the menu button set to the app's name, when being blank should be good enough.